packages/manager/modules/sms/src/sms/options/smppParameter/allowed-ips/allowed-ips.html

```diff
@@ -0,0 +1,88 @@
+<div class="col-sm-4">
+    <oui-tile
+        data-heading="{{ 'sms_options_smpp_parameter_allowed_ips_title' | translate }}"
+        data-loading="$ctrl.isLoading"
+    >
+        <oui-tile-definition>
+            <oui-tile-description>
+                <table class="oui-table">
+                    <tbody>
+                        <tr
+                            data-ng-repeat="ip in $ctrl.ips track by $index"
+                            class="oui-table__row"
+                        >
+                            <td class="oui-table__cell" data-ng-bind="ip"></td>
+                            <td class="oui-table__cell">
+                                <oui-button
+                                    size="s"
+                                    variant="secondary"
+                                    data-on-click="$ctrl.openModal('remove', ip)"
+                                >
+                                    <span class="oui-icon oui-icon-bin"></span>
+                                </oui-button>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </oui-tile-description>
+        </oui-tile-definition>
+        <oui-tile-definition>
+            <oui-tile-description>
+                <oui-button
+                    data-variant="secondary"
+                    data-on-click="$ctrl.openModal('add')"
+                    icon-right="oui-icon-plus"
+                    data-disabled="$ctrl.ips.length === $ctrl.MAX_ALLOWED_IP"
+                >
+                    <span
+                        data-translate="sms_options_smpp_parameter_allowed_ips_button_add"
+                        data-translate-values="{ ipCount: '(' + $ctrl.ips.length + '/' + $ctrl.MAX_ALLOWED_IP + ')' }"
+                    >
+                    </span>
+                </oui-button>
+            </oui-tile-description>
+        </oui-tile-definition>
+    </oui-tile>
+</div>
+<div data-ng-if="$ctrl.isOpenModal">
+    <div class="modal d-block">
```

Comment:
**nitpick:** `div` is a block element, `d-block` is redundant
```suggestion
    <div class="modal">
```

Reply:
It's need, because without, the modal don't display.